site/versions/master/docs/windows.md

```diff
@@ -81,16 +66,86 @@ $ ./bazel-bin/examples/java-native/src/main/java/com/example/myproject/hello-wor
 $ bazel run examples/java-native/src/main/java/com/example/myproject:hello-world
 ```
 
-### Build Python
 
-On Windows, we build a self-extracting zip file for executable python targets, you can even use
-`python ./bazel-bin/path/to/target` to run it in native Windows command line (cmd.exe).
-See more details in this [design doc](/designs/2016/09/05/build-python-on-windows.html).
+Building Bazel on Windows
+=========================
+
+We currently support only 64 bit Windows 7 or higher and we compile Bazel as a msys2 binary.
+
+To bootstrap Bazel on Windows, you will need:
+
+*    Java JDK 8 or later
+*    [Visual Studio](https://www.visualstudio.com/) (Community Edition is okay)
+*    [msys2](https://msys2.github.io/) (need to be installed at
+     ``C:\tools\msys64\``).
+*    Several msys2 packages. Use the ``pacman`` command to install them:
+     ``pacman -S gcc git curl zip unzip zlib-devel``
+
+To build Bazel:
+
+*    Open the msys2 shell.
+*    Clone the Bazel git repository as normal.
+*    Set the environment variables:
 
 ```bash
-$ bazel build examples/py_native:bin
-$ ./bazel-bin/examples/py_native/bin
-$ python ./bazel-bin/examples/py_native/bin    # This works in both msys and cmd.exe
-$ bazel run examples/py_native:bin
+export JAVA_HOME="$(ls -d C:/Program\ Files/Java/jdk* | sort | tail -n 1)"
+export BAZEL_SH=c:/tools/msys64/usr/bin/bash.exe
+```
+
+*     Run ``compile.sh`` in Bazel directory.
+*     If all works fine, bazel will be built at ``output\bazel.exe``.
+
+
+Maintaining Bazel Chocolatey package on Windows
```

Comment:
Sorry I am having second thoughts about this. Let's spilt these instructions into a separate `windows-chocolatey-maintenance.md` (this is not something an average Bazekl user on Windows needs to do)